app/lucy/src/app/components/Input/date-time-picker/date-time-picker.component.ts

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright © 2019 Province of British Columbia
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * **
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * **
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * File: time-picker.component.ts
+ * Project: lucy
+ * File Created: Monday, 27th January 2020 11:09:11 am
+ * Author: Williams, Andrea IIT (you@you.you)
+ * -----
+ * Last Modified: Monday, 27th January 2020 11:16:00 am
+ * Modified By: Williams, Andrea IIT (you@you.you>)
+ * -----
+ */
+
+import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
+import { NgbTimepicker, NgbTimeStruct } from '@ng-bootstrap/ng-bootstrap';
+import { FormMode } from 'src/app/models';
+import { MatDatepickerInputEvent } from '@angular/material';
+import * as moment from 'moment';
+import { Time } from '@angular/common';
+import { NgxMaterialTimepickerTheme } from 'ngx-material-timepicker';
+
+@Component({
+  selector: 'app-date-time-picker',
+  templateUrl: './date-time-picker.component.html',
+  styleUrls: ['./date-time-picker.component.css']
+})
+export class DateTimePickerComponent implements OnInit {
+
+  // Field header
+  @Input() header = '';
+  // Optional Input
+  @Input() editable = true;
+
+  get readonly(): boolean {
+    if (this.mode === FormMode.View) {
+      return true;
+    } else {
+      return !this.editable;
+    }
+  }
+
+  ///// Form Mode
+  private _mode: FormMode = FormMode.View;
+  // Get
+  get mode(): FormMode {
+    return this._mode;
+  }
+  // Set
+  @Input() set mode(mode: FormMode) {
+    this._mode = mode;
+  }
+  ////////////////////
+
+  private _dateTime: Date;
+  private _time: string;
+  dateTimeAsString: string;
+
+  // custom theme for the time picker
+  timePickerTheme: NgxMaterialTimepickerTheme = {
```

Comment:
I'm deleting this section since it's not used

Reply:
Oh yeah! We can remove that. I tried but I wasn't able to customize the clock 😞